xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -1415,7 +1415,7 @@ bool CJSONServiceDescription::prepareDescription(std::string &description, CVari
 
   if (description.at(0) != '{')
   {
-    description = StringUtils::Format("{%s}", description.c_str());
+    description = StringUtils::Format("{{{:s}}}", description);
```

Comment:
yeah {{ escapes one {, I had to change this because the syntax {%s} made cppformat think it was meant for it and threw an exception on invalid formatr string